docusaurus.config.js

```diff
@@ -16,103 +16,137 @@ const config = {
   favicon: "img/favicon.ico",
   organizationName: "unlock-protocol", // Usually your GitHub org/user name.
   projectName: "docs", // Usually your repo name.
-  plugins: [require.resolve("@cmfcmf/docusaurus-search-local"),
-  [
-    require.resolve('docusaurus-gtm-plugin'),
-    {
-      id: 'GTM-PRCCFV9', // GTM Container ID
-    }
+  plugins: [
```

Comment:
These changes are annoying. We should try to find a way to use the same format for everyone... linter maybe?